server/engine/Game.js

```diff
@@ -0,0 +1,230 @@
+const Board = require("./Board");
+const { Team, TeamColor } = require("./Team");
+const Operative = require("./Operative");
+const SpyMaster = require("./SpyMaster");
+const GameTurns = require("./GameTurns");
+const PlayerRoles = require("./PlayerRoles");
+const WordRoles = require("./WordRoles");
+
+class Game {
+  constructor(machId) {
+    this.machId = machId;
```

Comment:
I'm guessing this is suppose to be matchId? Small typo, not a huge problem :)

Reply:
haha, yeah. Thanks